x-pack/plugins/security_solution/public/common/components/visualization_actions/lens_attributes/users/kpi_total_users_area.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { LensAttributes } from '../../types';
+
+export const kpiTotalUsersAreaLensAttributes: LensAttributes = {
+  description: '',
+  state: {
+    datasourceStates: {
+      indexpattern: {
+        layers: {
+          '416b6fad-1923-4f6a-a2df-b223bb287e30': {
```

Comment:
hey could you explain to me what this file is about. where do these values come from? 🤔